README.md

```diff
@@ -166,6 +166,8 @@ The following Helm default values may be overridden:
 ```yaml
 clusterWideInstall: false
 serviceAccountName: cass-operator
+clusterRoleName: cass-operator-cr
```

Comment:
The "cass-operator" name for a cluster-role will already be taken if the helm chart has clusterWideInstall=true.  Therefore I added the -cr and -crb suffixes.